visualisation.md

```diff
@@ -1,5 +1,59 @@
 ---
-title: Beneficial Ownership Visual Language
+title: Beneficial Ownership Visualisation Standard
 layout: visualisation
 ---
 
+**The Beneficial Ownership Visualisation Standard (BOVS) is a set of visualisation rules** for producing diagrams that represent beneficial ownership structures as clearly as possible.
+
+We encourage anyone who needs to visualise beneficial ownership situations to follow these visualisation rules. The rules are compatible with any brand, style or colour scheme, and are provided under the Apache Licence 2.0.
```

Comment:
Perhaps slightly softer "We encourage anyone" -> "We offer these to anyone building visualisations of beneficial ownership structure, to support choices over how to represent different aspect of ownership."